packages/pods-core/src/v2/podConfig/MarkdownPodConfig.ts

```diff
@@ -55,6 +58,21 @@ export function createRunnableMarkdownV2PodConfigSchema(): JSONSchemaType<Runnab
         nullable: true,
       },
       wikiLinkToURL: {
+        type: "boolean",
+        default: false,
+        nullable: true,
+      },
+      convertTagNotesToLinks: {
```

Comment:
No need to address this in this PR, but there's no way to set these options (or even know they exist) unless the user chooses to save the configuration and opens the config. It might be worth considering if we can make this more discoverable.